flang/test/Preprocessing/macro-expansion.F90

```diff
@@ -0,0 +1,12 @@
+! RUN: %flang -fc1 -emit-fir -cpp -DNVAR=2+1+0+0 -o - %s | FileCheck %s
```

Comment:
You should use `%flang -E` tests for preprocessing rather than relying on lowering/FIR.